iree/compiler/Conversion/CMakeLists.txt

```diff
@@ -26,3 +62,8 @@ iree_cc_library(
 )
 
 ### BAZEL_TO_CMAKE_PRESERVES_ALL_CONTENT_BELOW_THIS_LINE ###
+# TODO: For some reason, these dependencies are not being added automatically.
```

Comment:
Because of the following I guess?

https://github.com/google/iree/blob/87a45fbb67b82a0f688ea1dce2e8424d09027b4e/build_tools/bazel_to_cmake/bazel_to_cmake_converter.py#L137-L143

I'm not sure this is still relevant given we have `ALWAYSLINK` deprecated long time ago. @GMNGeoffrey

Reply:
Yes! we entirely dropped `ALWAYSLINK`. Therefore, I assume that it *should* work if those lines get dropped.

Reply:
I'm going to land it without updating bazel to cmake as it was somewhat tricky to find out that this was an actual problem (also at head but just getting lucky). Can follow up with a tool fix.